app/kudos/templates/transaction/send.html

```diff
@@ -165,4 +173,50 @@ <h3 class="font-weight-300">{% blocktrans %}<span id="new_username">the user</sp
     <a id="" class="btn btn-gc-purple" href="{% url 'kudos_marketplace' %}">⬅ {% trans "Start Over" %}</a>
   </div>
 </section>
+
+<script>
+function removeStoredUser(e) {
+  e.preventDefault();
+  localStorage.removeItem('current_receiver')
+  document.getElementById('username').innerHTML = '';
+  document.getElementById('clearSelectedUser').style.display = 'none'
+}
+
+{% if not user_json and username %}
+  setTimeout(function() {
+    _alert("Sorry, we can't find the user");
+  }, 1000);
+{% elif user_json and username %}
+  localStorage.setItem('current_receiver', JSON.stringify({
+    id:  {{user_json.id }},
+    avatar_id: {{ user_json.avatar_id }},
+    avatar_url: '{{ user_json.avatar_url }}',
+    preferred_payout_address: '{{ user_json.preferred_payout_address }}',
+    text: '{{ user_json.text }}',
+    date: new Date()
+  }));
+  document.getElementById('clearSelectedUser').style.display = 'block'
+{% else %}
+  let encoded_data = localStorage.getItem('current_receiver')
+  if (encoded_data !== null) {
+    let data = JSON.parse(encoded_data);
+    let current = new Date();
+    let past = new Date(Date.parse(data.date))
+
+    if ((current.getTime() - past.getTime()) / (60 * 60) > 1440) {
+      localStorage.removeItem('current_receiver')
+    } else {
+      let option = document.createElement('option')
+      option.value = data.id
+      option.setAttribute('avatar_id',  data.avatar_id)
```

Comment:
Theres a couple places in this PR with some spacing issues - can you take another look and clean these up? thanks!